File: app/src/main/java/io/github/sithengineer/motoqueiro/hardware/MiBandEmpty.java
package io.github.sithengineer.motoqueiro.hardware;

import io.github.sithengineer.motoqueiro.hardware.capture.MiBandData;
import rx.Observable;

public class MiBandEmpty implements HardwareObservable<MiBandData>, MiBandService {
  @Override public Observable<MiBandData> listen() {
    return Observable.empty();
  }
}

File: app/src/main/java/io/github/sithengineer/motoqueiro/hardware/MiBandEmpty.kt
package io.github.sithengineer.motoqueiro.hardware

import io.github.sithengineer.motoqueiro.hardware.capture.MiBandData
import io.reactivex.Observable

class MiBandEmpty : HardwareObservable<MiBandData>, MiBandService {
  override fun listen(): Observable<MiBandData> {
    return Observable.empty()
  }
}
